operator/v1/types_network.go

```diff
@@ -431,16 +431,16 @@ type OVNKubernetesConfig struct {
 	// v4InternalSubnet is a v4 subnet used internally by ovn-kubernetes in case the
 	// default one is being already used by something else. It must not overlap with
 	// any other subnet being used by OpenShift or by the node network. The size of the
-	// subnet must be larger than the number of nodes. The value cannot be changed
-	// after installation.
+	// subnet must be larger than the number of nodes. The value can be changed after
+	// installation.
```

Comment:
In the case we make things mutable that were not previously, we would normally just drop this statement
```suggestion
	// subnet must be larger than the number of nodes.
```

Reply:
agree @arghosh93 let's just drop that sentence altogether